cluster/gce/util.sh

```diff
@@ -220,6 +226,36 @@ function upload-server-tars() {
   SALT_TAR_URL="${salt_gs_url/gs:\/\//https://storage.googleapis.com/}"
 }
 
+# Figure out which binary use on the server and assure it is available.
+# If KUBE_VERSION is specified use binaries specified by it, otherwise
+# use local dev binaries.
+#
+# Assumed vars:
+#   PROJECT
+# Vars set:
+#   SERVER_BINARY_TAR_URL
+#   SALT_TAR_URL
+function tars_from_version() {
```

Comment:
This can actually be moved to a common location, too, along with the `KUBE_VERSION` regexs (maybe `cluster/common.sh`?). Again, these are basically produced by "Kubernetes Release Engineering" (y'know, the once every two week "volunteer".)